Splunk_TA_paloalto/default/transforms.conf

```diff
@@ -161,6 +161,10 @@ REGEX = ^(?<dest_hostname>[^/]*)/
 SOURCE_KEY = ContentType
 REGEX = ^(?<major_content_type>[^/]*)/
 
+[extract_url_domain]	
```

Comment:
I think this is already extracted in the `extract_dest_hostname` transform on line 132. Is this extraction doing something different?

Reply:
If the value of the source_key would be an FQDN, then the url_domain would be just the domain name. So, we have written the extraction for url_domain separately.

Reply:
I see, so this is just the domain, not the FQDN. I have merged this in so now we have both. Thanks!